src/Features/Core/Portable/Completion/Providers/AbstractObjectInitializerCompletionProvider.cs

```diff
@@ -84,15 +90,7 @@ private bool IsLegalFieldOrProperty(ISymbol symbol, ISymbol within)
             return symbol.IsWriteableFieldOrProperty();
         }
 
-        protected CompletionItem CreateItem(
-            Workspace workspace,
-            string displayText,
-            TextSpan filterSpan,
-            Func<CancellationToken, Task<ImmutableArray<SymbolDisplayPart>>> descriptionFactory,
-            Glyph? glyph)
-        {
-            return new CompletionItem(this, displayText, filterSpan, descriptionFactory, glyph, rules: ObjectInitializerCompletionItemRules.Instance);
-        }
+        private static readonly CompletionItemRules s_rules = CompletionItemRules.Create(enterKeyRule: EnterKeyRule.Never);
```

Comment:
Weird.  Why does object initializer completion override the normal enter key rules?